kernel/drivers/block/block_transport_host.c

```
@@ -16,6 +16,24 @@ typedef struct {
 
+static size_t host_cstr_len(const char *s, size_t max_len) {
+    size_t n = 0;
+    if (!s)
+        return 0;
+    while (n < max_len && s[n])
+        n++;
+    return n;
+}
+
+static void host_cstr_copy(char *dst, size_t dst_len, const char *src) {
+    if (!dst || dst_len == 0)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)